config.go

```diff
@@ -12,14 +12,15 @@ type Config struct {
 	AssumeRoleTransitiveTagKeys []string
 	CallerDocumentationURL      string
 	CallerName                  string
-	CredsFilename               string
 	DebugLogging                bool
 	IamEndpoint                 string
 	Insecure                    bool
 	MaxRetries                  int
 	Profile                     string
 	Region                      string
 	SecretKey                   string
+	SharedCredentialsFilename   string
+	SharedConfigFilename        string
```

Comment:
Given that the underlying AWS API allows [multiple shared credentials and config files to be specified](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#shared-credentials-and-configuration) and we are not trying to maintain backwards compatibility, do we want these 2 fields to be of type `[]string`?

Reply:
That's a good idea